.github/workflows/build-dev.yaml

```diff
@@ -42,8 +42,6 @@ jobs:
 
       - name: Run golangci-lint
         uses: golangci/golangci-lint-action@v3.7.0
-        with:
-          args: --enable gofmt --enable gofumpt --timeout 5m  --build-tags e2e
```

Comment:
this was removed in favour of having the linter configuration in the .golangci.yaml as that allows us to configure not just the set of linters but also configuration for each linter separately.